infra/lambda/alert_to_webhook/index.py

```
@@ -0,0 +1,71 @@
+        action = "Scale-Out (인스턴스 기동)"
+    elif "Terminate" in detail_type:
+        action = "Scale-In (인스턴스 종료)"
+    else:
+        action = detail_type or "이벤트"
+    text = f"**ASG 스케일 이벤트**\n{action}\nASG: {asg_name}\nInstanceId: {instance_id}"
+    return {"content": text} if is_discord_style() else {"text": text}
+
+
+def lambda_handler(event, context):
+    try:
+        if "Records" in event and event["Records"]:
```

Ruff (0.15.6)

[warning] Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)